src/main/java/teammates/sqllogic/core/AccountRequestsLogic.java

```diff
@@ -104,25 +105,30 @@ public List<AccountRequest> getPendingAccountRequests() {
     }
 
     /**
-     * Creates/resets the account request with the given email and institute such that it is not registered.
+     * Creates/resets the account request with the given email and institute such
+     * that it is not registered.
```

Comment:
This is not in `master`. Did the merge do this? It does not seem necessary. There are a few other random formatting changes within this file too.

I am fine with this though, if it is difficult to undo; they do not affect the functionality anyway.

Reply:
Yep this was some formatting issue on my local machine, fixed it alr. Thanks!